### DISCUSSION TITLE
Can we get a new option to hide desktop widgets?

### DISCUSSION THREAD

**Discussion by anilaggrawal** [jacklandrin/OnlySwitch#93] [Ideas]
Since Sonoma lets you put Widgets on your desktop, which can sometimes carry sensitive info, it would be a good idea to have an option to toggle widgets on and off. I see some open source apps have popped up to provide this service.

**Comment by jacklandrin** [comment]
Hi,
This feature has already been integrated into Evolution since version 2.4.3. You can find it -> Settings->Evolution->Evolution Gallery->Hide Desktop Widget. Just press the tile, this switch will be added into your Evolution list. Have fun!
